### DISCUSSION TITLE
GitHub Actions Manual Trigger from Issue / Pull Request UI

### DISCUSSION THREAD

**Discussion by pavel-mikula-sonarsource** [community/community#27061] [Actions]
<p>Hello,</p>
<p>I’d like to propose a new feature for <a href="http://www.github.com" rel="noopener nofollow ugc">www.github.com</a> UI related to Actions.</p>
<p>We’d like to trigger a GitHub Action from an issue UI or pull request UI in a <strong>easy</strong> single click way, that would be used many times a day by each user.</p>
<p>Let’s imagine a new <code>issues</code> and <code>pull_request</code> trigger type, something like:</p>
<pre><code class="lang-auto">on:
  issues:
    types: [manual_dispatch]
  pull_requests:
    types: [manual_dispatch]
</code></pre>
<p>UI on <a href="http://www.github.com" rel="noopener nofollow ugc">www.github.com</a> would recognize such a trigger and create a section in the right menu on the issue/pull_request page:</p>
<pre><code class="lang-auto">Reviewers...
Assignees...
Labels...
Projects...
Milestone...
Linked issues...
**Actions**
 * First action name - clickable link
 * Second action name - clickable link
Notifications...
</code></pre>
<p>Dispatched action would provide usual issue/pull_request payload directly.</p>
<p>As this should be very easy to use, no parameters are expected. Comments could be used instead to deal with parametrized actions.</p>
<p>I know I already can:</p>
<ul>
<li>Manually trigger a workflow from Actions tab. That’s not easy for our use case as context must be provided as a parameter and the UI is far away from where the user is.</li>
<li>Trigger on <code>labeled</code>/<code>unlabeled</code> event. That’s also not easy, as the list of labels is long and the label is preserved on the issue without any state meaning. It’s hard to run the same thing twice over time.</li>
<li>Trigger on <code>issue_comment</code>. That’s powerful, but still not easy enough for users as we need to train users to use the well known comments.</li>
</ul>

**Comment by Simran-B** [comment, marked as answer]
<p>Sounds like an interesting idea. For feature requests, you should use the <a href="https://support.github.com/contact/feedback?contact%5Bcategory%5D=actions" rel="noopener nofollow ugc">Feedback form for GitHub Actions</a>. That will allow you to directly interact with the support team, and make it more convenient for the engineering team to collect and categorize suggestions. This forum isn’t a good place for such requests unless you want to start a discussion with the community about it.</p>

**Comment by pavel-mikula-sonarsource** [comment]
<p>Done, thanks for the suggestion <a class="mention" href="/simran-b">@Simran-B</a>.</p>
